### PR TITLE
changing data type of 't' to int in lamb_update_phase1

### PR DESCRIPTION
**Pull request by access2rohit**
## Description ##
Fixing data type of `t` in lamb_update_phase1

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by access2rohit** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Comment by access2rohit** [comment]
@sxjscience @eric-haibin-lin  PR is ready for review

File: src/operator/optimizer_op-inl.h [hunk 1]
```diff
@@ -1639,8 +1639,8 @@ struct LambUpdatePhaseOneKernel {
     DType g = mean_data[i] / (square_root::Map(var_data[i]) + epsilon) + wd * weight_data[i];
 
     if (bias_correction) {
-      DType mean_hat = mean_data[i] / (1. - power::Map(beta1, t));
-      DType var_hat = var_data[i] / (1 - power::Map(beta2, t));
+      DType mean_hat = mean_data[i] / (1. - std::pow(beta1, t));
```
Comment (apeforest) [inline src/operator/optimizer_op-inl.h]: Can we still re-use the power::Map() data structure here with a new signature for Map()?

Comment (eric-haibin-lin) [reply src/operator/optimizer_op-inl.h]: @apeforest  i'm not sure mshadow allows mixed data type as inputs

**Pull request merged**